lib/archethic_web/controllers/api/transaction_payload.ex

```diff
@@ -49,6 +49,8 @@ defmodule ArchethicWeb.API.TransactionPayload do
 
   def to_map(%{changes: changes}, acc) do
     Enum.reduce(changes, acc, fn {key, value}, acc ->
+      value = if key == :authorizedKeys, do: format_authorized_keys(value), else: value
```

Comment:
since you are already doing format_authorized_keys, maybe it can take both key and value and pattern match on the specific key you need. I think it will be more readable.